main.cpp

```diff
@@ -173,143 +173,143 @@ struct CarWash //2)
+    2)woofer material
+    3)tweeter material
+    4)ported design
+    5)active design with rated power
 3 things it can do:
-    1)
-    2)
-    3)
+    1)active design can amplify a line level signal
+    2)can split the signal in 2 bands at crossover frequency
+    3)port can boost the low frequencies
  */
 
 /*
-2)
+2)car jumpstarter
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)current rating in amperes
+    2)battery technology
+    3)weight
+    4)short circuit protection
+    5)charge display leds
 3 things it can do:
-    1)
-    2)
-    3)
+    1)jumpstart a car 
+    2)power devices trough USB
+    3)display the battery charge level
  */
 
 /*
-3)
+3)midi controller
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)amount of keys
+    2)amount of knobs
+    3)display size
+    4)performance controls
+    5)weight 
 3 things it can do:
-    1)
-    2)
-    3)
+    1)send midi data when keys pressed
+    2)display current patch
+    3)transpose octave
  */
 
 /*
-4)
+4)air contitioning unit
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)portable
+    2)cooling power in BTU
+    3)noise level
+    4)heather option
+    5)directional fan
 3 things it can do:
-    1)
-    2)
-    3)
+    1)remove environment heat to control room temperature
+    2)heat a room
+    3)select the fan speed
  */
 
 /*
-5)
+5)USB interface
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)speed in kb per second
+    2)standard (1.0 or 2.0 for USB ) 
+    3)cable length 
+    4)connector type
+    5)current it can receive for powering the board
 3 things it can do:
-    1)
-    2)
-    3)
+    1)get the program to be stored in memory
+    2)report errors
+    3)transmit serial data back to computer
  */
 
 /*
-6)
+6)DA converters
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)amount of converters
+    2)bit depth
+    3)clocking speed
+    4)max output current
+    5)output impedance
 3 things it can do:
-    1)
-    2)
-    3)
+    1)play audio
+    2)provide a constant voltaje for controlling a device
+    3)drive a transistor 
  */
 
 /*
-7)
+7)AD converters
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)amount of converters
+    2)bit depth
+    3)sampling speed
+    4)input impedance
+    5)overcurrent protection
 3 things it can do:
-    1)
-    2)
-    3)
+    1)read a constant sensor voltage
+    2)read an audio signal
+    3)read a TTL binary signal
  */
 
 /*
-8)
+8)memory
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)size
+    2)type of memory (ram , eprom, rom, flash)
+    3)clock speed
+    4)slot for flash card
+    5)power consumption
 3 things it can do:
-    1)
-    2)
-    3)
+    1)store script
+    2)store data read from sensors
+    3)store status 
  */
 
 /*
-9)
+9)reset button
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1)size
+    2)location on the board
+    3)color
+    4)material
+    5)useful life
```

Comment:
`useful life` -> `number of clicks before failure`?

Reply:
yes, I agree